ruby/pom.xml

```diff
@@ -86,7 +86,7 @@
         <dependency>
             <groupId>com.google.protobuf</groupId>
             <artifactId>protobuf-java</artifactId>
-            <version>3.0.0-alpha-3</version>
+            <version>3.0.0-beta-3</version>
```

Comment:
Realy 3.0.0-beta-3, not 3.0.0-beta-4?

I could be wrong, but it does not fit well with the changes in the headline.